.github/workflows/pypi.yaml

```diff
@@ -126,7 +126,7 @@ jobs:
             # PyPI does not accept .sigstore artifacts and
             # gh-action-pypi-publish has no option to ignore them.
             - name: Remove sigstore signatures before uploading to PyPI
-              run: rm ./dist/*.sigstore
+              run: rm ./dist/*.sigstore.json
```

Comment:
@tiran With the change of the suffix to `.sigstore.json`, is this removal still needed? Will PyPI accept `.json` extension files as part of the publish?

Reply:
I don't know whether PyPI supports uploads of `.sigstore.json` or not.

I recommend to delay this change until 0.18.0 has been released. There is no way to test the correctness of the change with PyPI. If we get it wrong, we have to role another release until we get it right.